process_report/process_report.py

```diff
@@ -336,6 +336,7 @@ def validate_pi_names(dataframe):
 def apply_credits_new_pi(dataframe, old_pi_file):
     new_pi_credit_code = "0002"
     new_pi_credit_amount = 1000
+    EXCLUDE_SU_TYPES = ["OpenShift GPUA100SXM4", "OpenStack GPUA100SXM4"]
```

Comment:
Does it make senses to fill this from a csv or txt file in github? I am asking out of curiosity not saying we should change it.

Reply:
It should be filled in from https://github.com/cci-moc/nerc-rates once that is in place and integrated.

Reply:
@knikolla does it make sense for this to be one of the items in your new repo so that it can be tracked when what SU types were excluded and when/if that changes?

Reply:
I swear I sent my comment before I saw that answer :)